test/cpp/api/functional.cpp

```diff
@@ -917,6 +917,25 @@ TEST_F(FunctionalTest, Linear) {
   }
 }
 
+TEST_F(FunctionalTest, Embedding) {
+  const auto input = torch::tensor({{1,2,4,5}, {4,3,2,9}}, torch::kLong);
+  auto weight = torch::empty({10, 3});
+  torch::nn::init::normal_(weight);
+  auto y = F::embedding(input, weight);
+  auto y_exp = torch::embedding(weight, input.contiguous(), -1, false, false);
+  ASSERT_TRUE(torch::allclose(y, y_exp));
+}
+
+TEST_F(FunctionalTest, EmbeddingBag) {
+  const auto input = torch::tensor({1,2,4,5,4,3,2,9}, torch::kLong);
+  auto offsets = torch::tensor({0,4}, torch::kLong);
+  auto weight = torch::empty({10, 3});
+  torch::nn::init::normal_(weight);
+  auto y = F::embedding_bag(input, weight, EmbeddingBagOptions().mode(torch::kSum).offsets(offsets));
+  auto y_exp = std::get<0>(torch::embedding_bag(weight, input, offsets, false, 0, false, torch::Tensor()));
+  ASSERT_TRUE(torch::allclose(y, y_exp));
```

Comment:
It would also be awesome to add a "no-options" test case `F::embedding_bag(input, weight)` :D